### PR TITLE
CJC-414

### PR DESCRIPTION
**Pull request by ccaprio97**
## What’s changing?

Returning Etag/unique upload identifier from the api

## Why?

Nac ticket 414

## This PR has:

- [x] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [ ] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [ ] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

**Comment by ccaprio97** [comment]
Making PR because its ready. but cant merge yet. Havent figured out the postman request yet to test it

File: etl/src/Piipan.Etl/Piipan.Etl.Func.BulkUpload/BulkUpload.cs [hunk 1]
```diff
@@ -46,7 +47,7 @@ public class BulkUpload
         /// to the per-state storage account and write access to the per-state database.
         /// </remarks>
         [FunctionName("BulkUpload")]
-        public async Task Run(
+        public async Task<UploadResponse> Run(
```
Comment (bgfoshay) [inline etl/src/Piipan.Etl/Piipan.Etl.Func.BulkUpload/BulkUpload.cs:50]: Thought: I believe this should be Task<IActionResult>

File: etl/src/Piipan.Etl/Piipan.Etl.Func.BulkUpload/BulkUpload.cs [hunk 2]
```diff
@@ -60,12 +61,14 @@ public class BulkUpload
                     var participants = _participantParser.Parse(input);
                     BlobProperties blobProperties = await blobClient.GetPropertiesAsync();
                     await _participantApi.AddParticipants(participants,  blobProperties.ETag.ToString());
+                    return new UploadResponse(blobProperties.ETag);
```
Comment (bgfoshay) [inline etl/src/Piipan.Etl/Piipan.Etl.Func.BulkUpload/BulkUpload.cs:64]: Thought: This should be

                var response = new UploadResponse(blobProperties.ETag);
                return new JsonResult(response) { StatusCode = StatusCodes.Status200OK };

File: etl/src/Piipan.Etl/Piipan.Etl.Func.BulkUpload/Models/UploadResponse.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Piipan.Etl.Func.BulkUpload.Models
+{
+    public class UploadResponse
+    {
+        public ETag uploadId { get; set; }
```
Comment (bgfoshay) [inline etl/src/Piipan.Etl/Piipan.Etl.Func.BulkUpload/Models/UploadResponse.cs:12]: Issue: Should be Pascal case. Also, you need to add Json attributes to these properties. e.g.

[JsonProperty("data", Required = Required.Always)]

**Pull request closed without merging**